<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela</class>
 <widget class="QMainWindow" name="Janela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle de acesso - Laboratórios de Informática</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="faceCapture">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>16</y>
      <width>282</width>
      <height>240</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Acesso">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>120</y>
      <width>297</width>
      <height>81</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>[+] Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>240</y>
      <width>297</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>70</number>
     </property>
     <item>
      <widget class="QPushButton" name="AdicionarNovo">
       <property name="text">
        <string>Novo Usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Detectar">
       <property name="text">
        <string>Detectar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>40</y>
      <width>289</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="matricula_label">
       <property name="text">
        <string>Matrícula/CPF:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Matricula"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="fazerReserva">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>176</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fazer Reserva</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>faceCapture</zorder>
   <zorder>Acesso</zorder>
   <zorder>fazerReserva</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuListar_Reservas">
    <property name="title">
     <string>Reservas</string>
    </property>
    <addaction name="actionListar"/>
   </widget>
   <addaction name="menuListar_Reservas"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionListar">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
